rust_dev_preview/tls/src/main.rs

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/*
+Purpose
+
+Set up a connection using a minimum version of
+
+Shows how to use rustls, hyper-rustls, and webpki-roots to set the minimum TLS version to 1.3 for
+outgoing connections.
+A sample call is then made using TLS 1.3 to AWS Key Management System (KMS) for a proof of concept.
```

Comment:
Then, uses TLS 1.3 to make a sample call to AWS Key Management Service (AWS KMS) as proof of concept.